contrib/core-contract-tests/tests/pox-4/pox_StackAggregationIncreaseCommand.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  logCommand,
+  PoxCommand,
+  Real,
+  Stub,
+  Wallet,
+} from "./pox_CommandModel.ts";
+import { poxAddressToTuple } from "@stacks/stacking";
+import { expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+
+/**
+ * The `StackAggregationIncreaseCommand` allows an operator to commit
+ * partially stacked STX to a PoX address which has already received
+ * some STX (more than the `stacking minimum`).
+ * This allows a delegator to lock up marginally more STX from new
+ * delegates, even if they collectively do not exceed the Stacking
+ * minimum, so long as the target PoX address already represents at
+ * least as many STX as the `stacking minimum`.
+ * This command calls stack-aggregation-increase.
+ *
+ * Constraints for running this command include:
+ * - The Operator must have locked STX on behalf of at least one stacker.
+ * - The PoX address must have partial committed STX.
+ * - The Reward Cycle Index must be positive.
```

Comment:
The reward cycle index must be one of the returned indexes by the first aggregation commit tx for the pox address.

Reply:
Yes, I pick it from the list at the generator level, [here](https://github.com/stacks-network/stacks-core/pull/4652/files#:~:text=...-,committedRewCycleIndexesOrFallback,-%2C)